doc/mcuboot/conf.py

```diff
@@ -24,7 +24,7 @@
 
 project = "MCUboot"
 copyright = "2019-2024, Nordic Semiconductor"
-version = release = "1.10.0"
+version = release = "2.0.99"
```

Comment:
looks good now. @de-nordic Can you confirm?

Reply:
By sdk-nrf versioning, this is how we name the versoin. By the MCUboot it would be 2.1.0-dev (https://github.com/nrfconnect/sdk-mcuboot/blob/3c2f2ff12bc20625cd65730b6036d061de4da5f7/boot/zephyr/VERSION
)

Reply:
Discussed offline and updated as 2.0.99.